spec.emu

```diff
@@ -390,6 +390,34 @@ contributors: Google, Ecma International
     </emu-note>
   </emu-clause>
 
+  <emu-clause id="sup-temporal-calendardateextendedyear" type="abstract operation">
+    <h1>
+      CalendarDateExtendedYear (
+        _calendar_: a String,
+        _date_: a Temporal.PlainDateTime, Temporal.PlainDate, or Temporal.PlainYearMonth,
+      ): either a normal completion containing an integer or an abrupt completion
+    </h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>It performs implementation-defined processing to find the year for the date corresponding to _date_ in the context of the calendar represented by _calendar_ relative to a well-defined epoch year for that calendar.</dd>
+    </dl>
+    <p>
+        This definition supersedes the definition provided in "Temporal proposal", #sec-temporal-calendardateextendedyear.
+    </p>
+    <emu-alg>
+      1. Let _year_ be ? Get(_date_, *"year"*).
```

Comment:
This won't work, because the value returned from this op is supposed to be the value of `date.year`. Get the [[ISOYear]] slot instead.